### ISSUE TITLE
pre-trained model 'redditcc_base' missing

### ISSUE THREAD

**Issue by Linohong** [neukg/KAT-TSLF#2]
Hello, this is Lino Hong from NLP lab Sogang University, South Korea.

In order to run warmup.sh, following the README, the pretrained model (assigned to 
the global variable MODEL in warmup.sh) is necessary, just like shown below.

![image](https://user-images.githubusercontent.com/19821168/145033826-be9b5a20-7147-4c27-88c6-52fd8c8880da.png)

However, referring to the download links in your github,
only the bart_wikilm is given. 

Where can I get access to the files of 'redditcc_base' directory?

Thank you KAT-TSLF team for the good work.

Lino Hong.

**Comment by Linohong** [comment]
There are two files to download for the given pretrained models. 
I missed the redditcc_base.zip part!
